libs/utils/decorate.js

```diff
@@ -96,3 +96,22 @@ export function decorateTextOverrides(el, options = ['-heading', '-body', '-deta
     el.classList.remove(override);
   });
 }
+
+export function decorateIconStack(el) {
+  const ulElems = el.querySelectorAll('ul');
+  if (ulElems.length) {
```

Comment:
super nit: early return

Reply:
@chrischrischris I have made the requested changes.